.github/workflows/npm-publish.yml

```diff
@@ -32,35 +29,44 @@ jobs:
         with: 
           registry-url: ${{ env.NPM_REGISTRY }}
           scope: "${{ env.SCOPE }}"
-
+        env:
```

Comment:
@mweichert if you add an env var above the steps definition it will be made available to all steps in the workflow. So if the `GIT_WORKSPACE` env var was moved the same place as `NODE_AUTH_TOKEN` then it will be available in the entire job and you won't need to add it to every step

Reply:
Ah, I figured that there was a way. Thanks! I'll make that adjustment.